include/swift/AST/Attr.def

```diff
@@ -274,6 +274,9 @@ DECL_ATTR(_implements, Implements,
 
 DECL_ATTR(NSKeyedArchiveLegacy, NSKeyedArchiveLegacy, OnClass, 68)
 
+SIMPLE_DECL_ATTR(_staticInitializeObjCMetadata, StaticInitializeObjCMetadata,
+                 OnClass, 69)
```

Comment:
This one should be UserInaccessible to keep it out of code completion, and should also be NotSerialized.